qiita_pet/templates/study_description_templates/sample_template_tab.html

```diff
@@ -9,7 +9,7 @@
       {% end %}
     </select>
     <br/><br/>
-    <a class="btn btn-primary" onclick="process_sample_template();">Process sample template</a>
+    <a class="btn btn-primary" onclick="process_sample_template();">Process sample template</a> <a class="btn btn-primary" onclick="extend_sample_template();">Add samples to study</a>
```

Comment:
You got it: https://github.com/biocore/qiita/issues/903

On (Mar-04-15|18:35), Joshua Shorenstein wrote:
> > @@ -9,7 +9,7 @@
> >        {% end %}
> >      </select>
> >      <br/><br/>
> > -    <a class="btn btn-primary" onclick="process_sample_template();">Process sample template</a>
> > +    <a class="btn btn-primary" onclick="process_sample_template();">Process sample template</a> <a class="btn btn-primary" onclick="extend_sample_template();">Add samples to study</a>
> 
> I can do that for all buttons in a separate pull request. Can you open an issue?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/892/files#r25835493